src/main/java/uk/gov/companieshouse/web/accounts/util/Navigator.java

```diff
@@ -35,7 +35,7 @@ public static String getNextControllerRedirect(Class clazz, String... pathVars)
             throw new MissingAnnotationException("Missing @RequestMapping value on class " + nextControllerClass.toString());
         }
 
-        return UrlBasedViewResolver.REDIRECT_URL_PREFIX + new UriTemplate(mappings[0]).expand(pathVars);
+        return UrlBasedViewResolver.REDIRECT_URL_PREFIX + new UriTemplate(mappings[0]).expand((Object[]) pathVars);
```

Comment:
Do we need the `Object[]` cast here?

Reply:
This was added to silence IDE warnings regarding ambiguity in this method call—we could modify the method signature to accept a variable argument parameter of type `Object...` but this might obscure the actual underlying `String` type expected. For the sake of silencing the warnings, I think this may be the better option.